src/routes/user.js

```diff
@@ -3,24 +3,44 @@
 const express = require('express');
 const userRouter = express.Router();
 
-const { users } = require('../models/index');
+const { users, db } = require('../models/index');
 const bearer = require('../middleware/bearer');
-const basic = require('../middleware/basic');
 
-// GET User - User Reads 
-userRouter.get('/user', async(req, res) => {
-  const userData = await users.findOne({ where: {username: req.body.username}});
-  res.status(200).json(userData);
+// GET User - User Reads
+// TODO: Return back all created Users
+userRouter.get('/users', async(req, res) => {
+  try {
+    const userData = await users.findOne({ where: {username: req.body.username}});
```

Comment:
instead of req.body.username you should use user that is added via the bearer token. Currently anyone with a token can update anyones like with this implementation.